src/pages/index.js

```diff
@@ -1,88 +1,109 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { graphql } from 'gatsby';
-import { Helmet } from 'react-helmet';
+import HubspotForm from 'react-hubspot-form';
 import jsYAML from 'js-yaml';
+import $ from 'jquery';
+import GitHubButton from 'react-github-btn';
+
+import 'slick-carousel/slick/slick.less';
 
 import EasyLayout from '../components/EasyLayout';
 
 import ImgBirdOnBricks         from '../assets/images/bird-on-bricks.svg';
-import ImgDebugLocally         from '../assets/images/debug-locally.svg';
-import ImgInstantFeedback      from '../assets/images/instant-feedback.svg';
-import ImgRealisticEnvironment from '../assets/images/realistic-environment.svg';
-import ImgMinimalResources     from '../assets/images/minimal-resources.svg';
-import ImgYourTools            from '../assets/images/your-tools.svg';
 import ImgQuote                from '../assets/images/quote.svg';
 import ImgVizualaiLogo         from '../assets/images/vizualai-logo.jpg';
 
+import SvgDebugLocally         from '../assets/images/debug-locally.inline.svg';
+import SvgInstantFeedback      from '../assets/images/instant-feedback.inline.svg';
+import SvgRealisticEnvironment from '../assets/images/realistic-environment.inline.svg';
+import SvgMinimalResources     from '../assets/images/minimal-resources.inline.svg';
+import SvgYourTools            from '../assets/images/your-tools.inline.svg';
+
 export default function HomePage({ location, data }) {
   const variables = jsYAML.safeLoad(data.file.internal.content);
 
+  useEffect(() => {
+    $(document).ready(function() {
+      changeTab($('.about-tabs li.tab:first-child').data('tabid'));
+
+      $('.about-tabs li.tab').click(function() {
+        changeTab($(this).data('tabid'));
+      });
+
+      function changeTab(tabId) {
+        $('.about-tabs li.tab').removeClass('active');
+        $('.about-tabs li.tab[data-tabId="'+tabId+'"]').addClass('active');
+        $('.tab-content div').hide();
+        $('#' + tabId).show();
+      }
+
+      import('slick-carousel').then(() => {$('.quotes').slick();});
```

Comment:
This import is inside the useEffect because of the SSR? There's a wrapper for slick-carousel that allows more natural imports on react components https://github.com/akiran/react-slick

Reply:
I couldn't get react-slick to work with SSR.  Maybe I could have with loadable-components?